mipengine/node_tasks_DTOs.py

```diff
@@ -87,6 +87,64 @@ class TableData(ImmutableBaseModel):
     ]
 
 
+class CommonDataElement(ImmutableBaseModel):
+    code: str
+    label: str
+    sql_type: str
+    is_categorical: bool
+    enumerations: Optional[Dict[str, str]] = None
+    min: Optional[float] = None
+    max: Optional[float] = None
+
+    def __eq__(self, other):
+        return (
+            isinstance(other, CommonDataElement)
+            and self.code == other.code
+            and self.label == other.label
+            and self.sql_type == other.sql_type
+            and self.is_categorical == other.is_categorical
+            and self.enumerations == other.enumerations
+            and self.max == other.max
+            and self.min == other.min
+        )
+
+
+class CommonDataElements(BaseModel):
+    values: Dict[str, CommonDataElement]
```

Comment:
It holds cdes and the values should be named cdes.
But in order to avoid the case:
cdes = CommonDataElements()
cdes.cdes
I named the field values to be more readable (cdes.values)